### DISCUSSION TITLE
Need Figma that was purchased ASAP!!!

### DISCUSSION THREAD

**Discussion by blattimore** [primefaces/community#1799] [PrimeVue]
Hello: 

We purchased the PrimeVue template and was supposed to have access to the figma. Right now we do not have access to this file: https://www.figma.com/file/ijQrxq13lxacgkb6XHlLxA/Preview-%7C-Ultima-2022?node-id=354%3A7715&t=gjWHprUDE5RJIg78-1

Please respond right away as we have a critical deadline and the figma is crucial. Why did we not get access for something that was paid for???

**Comment by onursenture** [comment]
Hi, Did you check your PrimeStore panel where you can download the Ultima's Figma file? https://www.primefaces.org/store/templates.xhtml

<img width="663" alt="image" src="https://github.com/primefaces/community/assets/244957/3f26ef90-d2d9-4c96-805f-606db62b9c05">

The link you shared and said you don't have access to is the preview file of Ultima's Figma file. We don't give anyone access to this file since it's our internal file to showcase the Figma file to those who review templates and consider purchases.
